wallet/wallet-node-client/tests/call_tests.rs

```diff
@@ -51,27 +51,7 @@ pub async fn start_subsystems(
 
     let p2p_config = p2p::config::P2pConfig {
         bind_addresses: vec!["127.0.0.1:0".to_owned()],
-        socks5_proxy: Default::default(),
-        disable_noise: Default::default(),
-        boot_nodes: Default::default(),
-        reserved_nodes: Default::default(),
-        max_inbound_connections: Default::default(),
-        ban_threshold: Default::default(),
-        ban_duration: Default::default(),
-        outbound_connection_timeout: Default::default(),
-        ping_check_period: Default::default(),
-        ping_timeout: Default::default(),
-        max_clock_diff: Default::default(),
-        node_type: Default::default(),
-        allow_discover_private_ips: Default::default(),
-        msg_header_count_limit: Default::default(),
-        msg_max_locator_count: Default::default(),
-        max_request_blocks_count: Default::default(),
-        user_agent: common::primitives::user_agent::mintlayer_core_user_agent(),
-        max_message_size: Default::default(),
-        max_peer_tx_announcements: Default::default(),
-        max_unconnected_headers: Default::default(),
-        sync_stalling_timeout: Default::default(),
+        ..p2p::config::P2pConfig::default()
```

Comment:
Please don't change these instances. We deliberately keep the list so that it has a side effect on every change and makes developers react to compiler error to make the decision each time.